### PR TITLE
Fence links generated in Bond

### PR DESCRIPTION
**Pull request by gpolumbo-broad**
- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/cljs/main/broadfcui/page/profile.cljs [hunk 1]
```diff
@@ -38,21 +38,10 @@
...
-       client_id
-       "&scope=openid+google_credentials&redirect_uri="
-       (js/encodeURIComponent
-        (let [loc js/window.location]
-          (str (.-protocol loc) "//" (.-host loc) "/#fence-callback")))
-       "&state=" (encode-base64-json {:provider provider})))
+(defn auth-url-link [href text]
+  (if href
+    (links/create-external {:href href :target "_self"} text)
+    (spinner {:ref "pending-spinner"} "Getting link information...")))
```
Comment (zarsky-broad) [inline src/cljs/main/broadfcui/page/profile.cljs:44]: [String refs are deprecated](https://reactjs.org/docs/refs-and-the-dom.html#legacy-api-string-refs), but given that there are lots of others you could conceivably leave this in. Just something to be aware of.

File: src/cljs/main/broadfcui/utils.cljs [hunk 2]
```diff
@@ -164,3 +164,9 @@
   (.getElementById js/document "app"))
 
 (def build-timestamp (generate-build-timestamp))
+
+(defn decode-base64-json [b64string]
+  (parse-json-string (js/window.atob b64string) true))
+
+(defn encode-base64-json [map]
+  (js/window.btoa (->json-string map)))
```
Comment (zarsky-broad) [inline src/cljs/main/broadfcui/utils.cljs:172]: Nit: newline

**Review by zarsky-broad — APPROVED** [review]

**Review by ansingh7115 — APPROVED** [review]
Looks good to me!

**Comment by gpolumbo-broad** [comment]
jenkins retest

**Comment by gpolumbo-broad** [comment]
jenkins retest please

**Pull request merged**